### PR TITLE
Test ok disabled for now

### PR DESCRIPTION
**Pull request by shadrackkibet**
@dannyparsons .This is ready for now.

### CODE REVIEW

**Comment by lilyclements** [comment]
I'm going to edit the layout of this first, and then Danny can merge it from my branch. If that's okay!

**Comment by shadrackkibet** [comment]
@lilyclements .That is okay.

File: instat/dlgSummaryTables.vb [hunk 1]
```diff
@@ -35,12 +36,24 @@ Public Class dlgSummaryTables
         End If
     End Sub
 
+    Private Sub TestOkEnabled()
+        If Not ucrReceiverFactor.IsEmpty AndAlso Not ucrReceiverVariate.IsEmpty Then
+            ucrBase.OKEnabled(True)
+        Else
+            ucrBase.OKEnabled(False)
+        End If
+        ucrBase.OKEnabled(False)
```
Comment (dannyparsons) [inline instat/dlgSummaryTables.vb]: Great, can you now comment out the lines above and also add a comment in the code to say that this is temporary?

Then you need to merge in the changes from Lily's branch where she did work on the layout.

**Comment by shadrackkibet** [comment]
@dannyparsons .This is ready for you now.

**Pull request merged**